lib/activity.js

```diff
@@ -54,8 +54,8 @@ export const getActivity = async () => {
 export const maybeCreateActivityFile = () => maybeCreateFile(paths.activity)
 
 export const formatActivityObject = ({ type, message, timestamp }) => {
-  return formatLog(
-    `${type.toUpperCase().padEnd(5)} ${message}`,
-    new Date(timestamp)
-  )
+  return formatLog(`${type.toUpperCase().padEnd(5)} ${message}`, {
+    date: new Date(timestamp),
```

Comment:
Are we parsing `timestamp` from a `String` to a `Date` because we converted it from a `Date` to a `String` on line 33 above? Maybe we can avoid that roundtrip?

Reply:
fixed